basics/section_2-23.rb

```diff
@@ -0,0 +1,42 @@
+def multiply(first_num, second_num)
+  # last evaluated expression of the method is implicitly returned
+  # we can also explicitly use the `return` keyword
+  first_num * second_num
+end
+
+puts multiply(2, 5)
+
+############
+
+def operator(first_num, second_num, operation)
+  if operation == "multiply"
+    first_num * second_num
+  elsif operation == "divide"
+    first_num / second_num
```

Comment:
The reason I suggest this instead is because:
```
1/2 == 0
```
This would mean that you expect the consumer of this method to do the conversion to `.to_f` first before passing the value in. This would mean you need to let the consumer of this method know in whatever way you can so that bugs can be prevented.

If we do as per my suggestion, this concern is nonexistent. consumer of this method can pass in int or float, it will process correctly